CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_policy(SET CMP0048 NEW)  # project_VERSION* variables populated from project(... VERSION x.x.x) string
+
+project(Libint
+        VERSION 1.2.0
+        LANGUAGES C CXX)
+set(Libint_AUTHORS      "Edward F. Valeev and Justin T. Fermann")
+set(Libint_DESCRIPTION  "Parallel implementation of the Effective Fragment Potential (EFP) method")
```

Comment:
replace with "Library for the evaluation of molecular integrals of two-body
operators over Gaussian functions"